File: packages/sdk-ts/src/manifests/bombora.ts
import { ToolManifest, Permission } from '../manifest.js';

export const bomboraManifest = new ToolManifest({
  connector: 'bombora',
  description: 'Bombora Intent Data API',
  tools: {
    get_surge_scores: Permission.READ,
    get_topic_clusters: Permission.READ,
    get_weekly_report: Permission.READ,
    search_companies: Permission.READ,
  },
});

File: packages/sdk-py/src/grantex/manifests/bombora.py
from grantex.manifest import ToolManifest, Permission

manifest = ToolManifest(
    connector="bombora",
    description="Bombora Intent Data API",
    tools={
        "get_surge_scores": Permission.READ,
        "get_topic_clusters": Permission.READ,
        "get_weekly_report": Permission.READ,
        "search_companies": Permission.READ,
    },
)
